starboard/elf_loader/BUILD.gn

```diff
@@ -161,8 +161,8 @@ if (sb_is_evergreen_compatible) {
 }
 
 # TODO: b/309493306 - Stop building evergreen targets for all non-evergreen platforms.
-if (!is_host_win) {
-  target(gtest_target_type, "elf_loader_test") {
+if (current_toolchain == starboard_toolchain && !is_host_win) {
+  target(starboard_level_gtest_target_type, "elf_loader_test") {
```

Comment:
note that making this a "starboard_level_gtest_target_type" it will be built as a single binary : elf_loader_test.exe
I dont think that approach works for evergreen

Reply:
This should have always been built with starboard only. Everything from loader and crashpad side should never be built with  Cobalt toolchain